tests/Yaapii.Atoms.Tests/IO/ZipTest.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using System.IO;
+
+namespace Yaapii.Atoms.IO.Tests
+{
+    public class ZipTest
+    {
+        [Fact]
+        public void ReturnsStreamGreater0()
+        {
+            string folderPath = Path.GetTempPath() + "\\" + "ZipTestFolder";
```

Comment:
You can use Path.GetDirectoryName("ZipTests") - that will result in a path which is inside the build directory (because build dir is the current working directory).